hyp3_autorift/process.py

```diff
@@ -178,6 +178,10 @@ def process(reference: str, secondary: str, parameter_file: str = DEFAULT_PARAME
         gdal.SetConfigOption('GDAL_DISABLE_READDIR_ON_OPEN', 'EMPTY_DIR')
         gdal.SetConfigOption('AWS_REQUEST_PAYER', 'requester')
         gdal.SetConfigOption('AWS_REGION', 'eu-central-1')
+        # Also set for new CXX threads in Geogrid/autoRIFT
```

Comment:
```suggestion
        # Also set for CXX threads in Geogrid/autoRIFT
```
Might not be new next time we look at it

Reply:
gdal is configured at initialization, and new threads are configured independently, so the environment variable is for *new* threads (new is modifying CXX threads not Geogrid/autoRIFT) -- we need to set it for both current threads and new threads.